auth0/v3/management/custom_domains.py

```diff
@@ -0,0 +1,80 @@
+from .rest import RestClient
+
+
+class CustomDomains(object):
+
+    """Auth0 custom domains endpoints
+
+    Args:
+        domain (str): Your Auth0 domain, e.g: 'username.auth0.com'
+
+        token (str): Management API v2 Token
+
+        telemetry (bool, optional): Enable or disable Telemetry
+            (defaults to True)
+    """
+
+    def __init__(self, domain, token, telemetry=True):
+        self.domain = domain
+        self.client = RestClient(jwt=token, telemetry=telemetry)
+
+    def _url(self, id=None):
+        url = 'https://%s/api/v2/custom-domains' % self.domain
+        if id is not None:
+            return url + '/' + id
+        return url
+
+    def all(self, extra_params=None):
```

Comment:
this endpoint doesn't accept extra params, at least by reading the api explorer. Would you remove it please?